implementations/rust/ockam/ockam_core/src/vault/mod.rs

```diff
@@ -0,0 +1,37 @@
+//! Core types and traits of the Ockam vault.
+//!
+//! This crate contains the core types and traits of the Ockam vault and is intended
+//! for use by other crates that either provide implementations for those traits,
+//! or use traits and types as an abstract dependency.
+#![deny(unsafe_code)]
```

Comment:
I think 12 lines below should be deleted, as all of this should be already present in ockam_core/src/lib.rs